README.md

```diff
@@ -1,3 +1,5 @@
+![Logo](https://raw.githubusercontent.com/romanz/electrs/master/logo/logo.svg)
```

Comment:
Is there any issue with relative path? Absolute will look broken until merged.

Reply:
TIL relative paths work for images :)